app/helpers/catalog_helper.rb

```diff
@@ -257,6 +270,19 @@ def catalog_tab_conditions(record)
     }
   end
 
+  def catalog_tab_edit_conditions(record)
+    resource = request = detail = false
+    detail = !detail if record[:display]
```

Comment:
The way detail is set here is weird - think this is clearer:

```suggestion
    resource = request = false
    detail = !!record[:display]
```

Reply:
If you're not familiar with it `!!` is a double-negation, so in Ruby that basically is shorthand for converting an object to a boolean.

```ruby
> !!true
# => true
> !!false
# => false
> !!nil
# => false
> !!Object.new
# => true
```